py/torch_tensorrt/dynamo/compile.py

```diff
@@ -201,6 +198,17 @@ def compile_module(
 
         submodule = getattr(partitioned_module, name)
 
+        # Store the input nodes for this TRT subgraph
+        submodule_input_nodes = []
+        submodule_node = None
+        for node in partitioned_module.graph.nodes:
+            if node.name == name:
+                submodule_node = node
+                if len(node.args) > 0:
+                    for i in range(len(node.args)):
+                        if node.args[i].op == "placeholder":
+                            submodule_input_nodes.append(node.args[i])
+
```

Comment:
Add a `break` here after the submodule inputs are populated, since there should only ever be one node with this submodule name.